common/wscript

```diff
@@ -142,6 +142,7 @@ def build(bld):
     clientlib.defines = 'HAVE_CONFIG_H'
     clientlib.use = uselib
     if bld.env['IS_WINDOWS']:
+         clientlib.env['cxxshlib_PATTERN'] = 'lib%s.dll'
```

Comment:
It changes some filenames; jack/profiler.dll to jack/libprofiler.dll and jack/netmanager.dll to jack/libnetmanager.dll.
Which is better?

Apart from it, I noticed that my patch has two problems:
1. Calling FreeLibrary() in locate_dll_driver() should be removed.
2. Windows audio device name sometimes longer than JACK_CLIENT_NAME_SIZE. It causes a device name cannot be specified.

In my understanding, changing JACK_CLIENT_NAME_SIZE breaks ABI compatibility. So I changed to use long name in the small part of PortAudio specific code and use truncated name for most common Jack code.

I'm sorry for insufficient tests. I'll fix the pull request.